_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Showcase
+  blog: Blog
+  community: Community
+  assets: Assets
+  download: Download
+  docs: Docs
+  contribute: Mitwirken
+  donate: Spenden
+  language: Sprache
+footer:
+  download: Download
+  documentation: Dokumentation
+  features: Features
+  web_editor: Web Editor
```

```suggestion
  web_editor: Online-Editor
```
I know this looks like swapping English with English, but that is more common I swear haha